azext_edge/tests/edge/orchestration/resources/connector/opcua/test_opcua_certs_client_unit.py

```diff
@@ -231,20 +237,55 @@ def test_client_add(
             {},
             "/fake/path/pubkey.der",
             "/fake/path/prikey.pem",
+            "subjectname",
+            "uri",
             "Public key file name pubkey and private key file name prikey must match.",
         ),
+        # subject name not found
+        (
```

Comment:
personally I like using dicts for params, especially when the tuples (like here) get very long.

a dict should help with avoiding incorrect order/missing params.

Reply:
yeah good idea, will add this to a TODO, since this would be a bigger change also include the other 2 test files